doc/src/manual/arrays.md

```diff
@@ -49,26 +49,26 @@ or a series of dimension sizes passed as a variable number of arguments.
+| [`falses(A)`](@ref)                         | a `BitArray` with all values `false` and the same shape as `A`                                                                                                                                                                                           |
+| [`reshape(A, dims...)`](@ref)               | an array with the same data as `A`, but with different dimensions                                                                                                                                                                                        |
 | [`copy(A)`](@ref)                           | copy `A`                                                                                                                                                                                                                                                 |
 | [`deepcopy(A)`](@ref)                       | copy `A`, recursively copying its elements                                                                                                                                                                                                               |
 | [`similar(A, element_type, dims...)`](@ref) | an uninitialized array of the same type as the given array (dense, sparse, etc.), but with the specified element type and dimensions. The second and third arguments are both optional, defaulting to the element type and dimensions of `A` if omitted. |
-| [`reinterpret(T, A)`](@ref)                 | an array with the same binary data as the given array, but with the specified element type                                                                                                                                                               |
-| [`rand(dims)`](@ref)                        | [`Array`](@ref) of `Float64`s with random, iid [^1] and uniformly distributed values in the half-open interval ``[0, 1)``                                                                                                                                |
-| [`randn(dims)`](@ref)                       | [`Array`](@ref) of `Float64`s with random, iid and standard normally distributed random values                                                                                                                                                           |
+| [`reinterpret(T, A)`](@ref)                 | an array with the same binary data as `A`, but with the specified element type `T`                                                                                                                                                                      |
+| [`rand(dims...)`](@ref)                     | an array of `Float64`s with random, iid [^1] and uniformly distributed values in the half-open interval ``[0, 1)``                                                                                                                                   |
```

Comment:
Restore the "[`Array`](@ref)", which provides meaningful information? (`rand(dims...)` doesn't return some unspecific `<:AbstractArray`, it returns an `Array`.)

A step further, perhaps "an [`Array`](@ref)`{Float64}` with shape `dims` and random, iid [^1] values drawn uniformly from the half-open interval ``[0, 1)``"? ( --m odulo how one handles that `@ref` correctly.)